definitions/checks/check_tmout.rb

```diff
@@ -0,0 +1,16 @@
+class Checks::CheckTmout < ForemanMaintain::Check
+  metadata do
+    label :check_tmout_variable
+    description 'Check if TMOUT environment variable is set'
+    tags :pre_upgrade
+  end
+
+  def run
+    assert(tmout_set?, 'The TMOUT environment variable is set on system.'\
+          " Run 'unset TMOUT' command to unset this variable.")
+  end
+
+  def tmout_set?
+    execute('printenv TMOUT').empty?
```

Comment:
Is there any difference from using ENV['TMOUT']?
Is the condition correct? empty? returns true when the var is not set, right?
Also consider TMOUT=0 as valid way to turn off auto-logout.